pkg/services/team/model.go

```diff
@@ -90,13 +90,6 @@ type SearchTeamsQuery struct {
 	HiddenUsers  map[string]struct{}
 }
 
-type ListTeamsCommand struct {
```

Comment:
This is removing the first attempt to expose teams+user list in:
https://github.com/grafana/grafana/pull/90418/files#diff-adf419aff0a45d3551e2a162d072b71e887abf6e06e2ef556c7125df131bf112

Rather than merging this with the existing services, we can make an explicit version with fewer compromises